src/cmd/manage_nsfs.js

```diff
@@ -331,20 +331,25 @@ async function manage_bucket_operations(action, data, argv) {
     }
 }
 
-async function account_management(argv, from_file) {
+async function account_management(argv, path_to_json_options) {
+    const type = argv._[0] || '';
     const action = argv._[1] || '';
     const show_secrets = Boolean(argv.show_secrets) || false;
-    const data = await fetch_account_data(argv, from_file);
+    let user_input;
+    // currently we use from_file only in add action
+    if (path_to_json_options) user_input = await get_options_from_file(type, action, path_to_json_options);
```

Comment:
looks much better!
I think that we should call this function at the same time we call validate_flags_arguments() function and integrate the 2 functions together, so that the validations will be called once

Reply:
I'm not sure I'm following... `validate_flags_arguments` is called in the main only once and it validates the command (that we have the type and action, that we don't have extra flags, and the values are according to the types we allowed).

In this PR we validate the command, but we also need to validate the options inside the JSON - and we are reusing the same functions as in `validate_flags_arguments`.

Reply:
yes, my point here is that we need to validate the arguments at the same time, regardless of their origin (flags/file)
so if flags are checked on the main function, I don't see a reason for checking the arguments here (if they originated from a file)

Reply:
I will move it to the main function.